.openpublishing.redirection.json

```diff
@@ -528,6 +528,10 @@
         {
             "source_path": "dotnet-desktop-guide/framework/wpf/properties/custom-dependency-properties.md",
             "redirect_url": "/dotnet/desktop/wpf/advanced/custom-dependency-properties?view=netframeworkdesktop-4.8"
+        },
+        {
+            "source_path": "dotnet-desktop-guide/net/winforms/whats-new/index.md",
+            "redirect_url": "/dotnet/desktop/winforms/whats-new/net60?view=netdesktop-5.0"
```

Comment:
This looks strange "net60?view=netdesktop-5.0"

Reply:
I need to do a PR in the repo that just mass updates these from 5.0 to 6.0. At the time the 6.0 moniker didn't exist.